Index: linux-gpib-kernel/Makefile
===================================================================
--- linux-gpib-kernel/Makefile    (revision 1914)
+++ linux-gpib-kernel/Makefile    (working copy)
@@ -5,6 +5,8 @@
 VERBOSE ?= 0
 ENABLE_PCMCIA ?= 0
 GPIB_DEBUG ?= 0
+BASE_PATH ?= /
+KERNEL_VERSION ?= `uname -r`

 all:
 		$(MAKE) -C $(LINUX_SRCDIR) V=$(VERBOSE) modules \
@@ -28,5 +28,5 @@
 		M="$(GPIB_SRCDIR)/drivers/gpib" \
 		GPIB_TOP_DIR=$(GPIB_SRCDIR) \
 		INSTALL_MOD_DIR=gpib
-	/sbin/depmod -A
+	depmod -A -b $(BASE_PATH) $(KERNEL_VERSION)
